sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -167,68 +167,123 @@ ConfigurationAsyncClient client = new ConfigurationClientBuilder()
         .connectionString(appConfigConnectionString)
         .buildAsyncClient();
 
-client.listSettings(new SettingSelector().setLabels(periodicUpdateLabel))
+client.listConfigurationSettings(new SettingSelector().setLabels(periodicUpdateLabel))
     .subscribe(setting -> updateConfiguration(setting));
 ```
 
 ## Examples
 
-The following sections provide several code snippets covering some of the most common Configuration Service tasks, including:
-- [Create a Configuration Setting](#create-a-Configuration-Setting)
-- [Retrieve a Configuration Setting](#retrieve-a-Configuration-Setting)
-- [Update an existing Configuration Setting](#update-an-existing-Configuration-Setting)
-- [Delete a Configuration Setting](#delete-a-Configuration-Setting)
-
-### Create a Configuration Setting
-
-Create a Configuration Setting to be stored in the Configuration Store. There are two ways to store a Configuration Setting:
-- addSetting creates a setting only if the setting does not already exist in the store.
-- setSetting creates a setting if it doesn't exist or overrides an existing setting.
+The following sections provide several code snippets covering some of the most common configuration service tasks, including:
+- [Create a configuration setting][sample_hello_world]
+- [Retrieve a configuration setting][sample_hello_world]
```

Comment:
Is this what you intended to link to? All these link to the hello_world sample. They appeared to be different anchors before to the actual section below.

Reply:
hello world example has all these operations.

Reply:
The anchors were there so the user wouldn't have to scroll to the appropriate header for that code snippet.

Reply:
Still has not been resolved.

Reply:
It should already resolved.